JASP-Engine/JASP/po/R-nl.po

```diff
@@ -950,6 +1379,115 @@ msgstr ""
+"on the <b>%5$s</b> distribution, the inherent risk <b>(%6$s)</b>, the "
+"control risk <b>(%7$s)</b> and the expected errors. Consequently, if the sum "
+"of errors from the audited observations remains below <b>%8$s</b>, the "
+"maximum misstatement is estimated to be below materiality."
+msgstr ""
+"De meest aannemelijke fout in de data was naar verwachting <b>%1$s</b>. De "
+"steekproefgrootte die nodig is voor een materialiteit van <b>%2$s</b>, ervan "
+"uitgaande dat de steekproef <b>%3$s</b> volledige fouten bevat, is <b>%4$s</"
+"b>. Deze steekproefgrootte is gebaseerd op de <b>%5$s</b> verdeling, het "
+"inherente risico <b>(%6$s)</b>, het controle risico <b>(%7$s)</b> en de "
+"verwachte fouten. Met als gevolg dat wanneer de som van fouten van de audit "
+"observaties beneden <b>%8$s</b> blijft, de maximale fout beneden "
+"materialiteit geschat wordt."
+
+msgid ""
+"The most likely error in the data was expected to be <b>%1$s</b>. The sample "
+"size that is required for a materiality of <b>%2$s</b>, assuming the sample "
+"contains <b>%3$s</b> full errors, is <b>%4$s</b>. This sample size is based "
+"on the <b>%5$s</b> distribution, the inherent risk <b>(%6$s)</b>, the "
+"control risk <b>(%7$s)</b> and the expected errors. The information in this "
+"prior distribution states that there is a <b>%8$s%%</b> prior probability "
+"that the population misstatement is lower than materiality. Consequently, if "
+"the sum of errors from the audited observations remains below <b>%9$s</b> "
+"the maximum misstatement is estimated to be below materiality."
+msgstr ""
+"De meest aannemelijke fout in de data was naar verwachting <b>%1$s</b>. De "
+"steekproefgrootte die nodig is voor een materialiteit van <b>%2$s</b>, ervan "
+"uitgaande dat de steekproef <b>%3$s</b> volledige fouten bevat, is <b>%4$s</"
+"b>. Deze steekproefgrootte is gebaseerd op de <b>%5$s</b> verdeling, het "
+"inherente risico <b>(%6$s)</b>, het controle risico <b>(%7$s)</b> en de "
+"verwachte fouten. De informatie in deze prior verdeling zegt dat er een <b>"
+"%8$s%%</b> prior waarschijnlijkheid is dat de populatie afwijking lager is "
+"dan de materialiteit. Met als gevolg dat wanneer de som van fouten van de "
+"audit observaties beneden <b>%9$s</b> blijft, de maximale fout geschat wordt "
+"beneden materialiteit te zijn."
+
+msgid "random"
+msgstr "willekeurig"
+
+msgid "fixed interval"
+msgstr "vast interval"
+
+msgid "cell"
+msgstr "cel"
+
+msgid ""
+"From the population of <b>%1$s</b> observations, <b>%2$s</b> observations "
+"were selected using a <b>%3$s record sampling</b> method. <br><b>Warning:</"
+"b> A monetary unit sampling method was tried but failed."
+msgstr ""
+"Van de populatie van <b>%1$s</b> observaties, waren <b>%2$s</b> observaties "
+"geselecteerd met <b>%3$s steekproef nemen</b> methode. <br><b>Waarschuwing:</"
+"b>  Een munteenheid steekproef neem methode lukte niet."
```

Comment:
Een munteenheid steekproef nemen is niet gelukt.

Reply:
Ok